ui/src/data_explorer/containers/DataExplorer.tsx

```diff
@@ -523,13 +523,23 @@ export class DataExplorer extends PureComponent<Props, State> {
       this.templates
     )
 
-    const queryDraft = {
-      query,
-      queryConfig,
-      source: source.links.self,
-      type: QueryType.InfluxQL,
+    if (this.service) {
```

Comment:
what do you think about naming this as a variable for self-documenting purposes?

Reply:
renamed to isFlux for clarity